themes/custom/az_barrio/az_barrio.theme

```diff
@@ -446,5 +450,25 @@ function az_barrio_preprocess_image(&$variables) {
     unset($variables['attributes']['height']);
     $variables['attributes']['class'] = ['card-img-top'];
   }
+
+}
+
+/**
+ * Implements hook_preprocess_block().
+ */
+function az_barrio_preprocess_block(&$variables) {
+  if ($variables['attributes']['id'] === 'block-az-barrio-sidebar-menu') {
+    $blockid = str_replace(['block-', '-'], ['', '_'], $variables['attributes']['id']);
+    $variables['content']['#attributes']['block'] = $blockid;
```

Comment:
During our 2020-01-08 Friday meeting @tadean pointed out that this adds a `block` attribute to the ul element in the rendered markup which is not ideal.